<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Aparna" failures="0" tests="2" name="com.test.testcases.AddCustomerTest" time="7.479" errors="1" timestamp="2022-09-17T21:41:44 EDT" skipped="0">
  <testcase classname="com.test.testcases.AddCustomerTest" name="addCustomer" time="3.540"/>
  <system-out>
    <![CDATA[Customer added]]>
  </system-out>
  <testcase classname="com.test.testcases.AddCustomerTest" name="addCustomer" time="3.939">
    <error message="Illegal char &lt;:&gt; at index 58: user.dir\target\error-screenshots\addCustomerSat Sep 17 21:41:43 EDT 2022.jpg" type="java.nio.file.InvalidPathException">
      <![CDATA[java.nio.file.InvalidPathException: Illegal char <:> at index 58: user.dir\target\error-screenshots\addCustomerSat Sep 17 21:41:43 EDT 2022.jpg
at java.base/sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:182)
at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:153)
at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
at java.base/sun.nio.fs.WindowsPath.parse(WindowsPath.java:92)
at java.base/sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:232)
at java.base/java.io.File.toPath(File.java:2387)
at org.apache.commons.io.FileUtils.doCopyFile(FileUtils.java:1390)
at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:885)
at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:835)
at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:802)
at com.test.utilities.TestUtils.captureScreenshot(TestUtils.java:25)
at com.test.listeners.CustomListeners.onTestFailure(CustomListeners.java:38)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:96)
at org.testng.internal.TestInvoker.runTestResultListener(TestInvoker.java:219)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:656)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- addCustomer -->
  <system-out/>
</testsuite> <!-- com.test.testcases.AddCustomerTest -->
